### ISSUE TITLE
Errors with the zipfiles, can't get the data for the course

### ISSUE THREAD

**Issue by morenobcn** [PacktPublishing/Learning-the-FOSS4g-Stack-Spatial-SQL-with-Postgres-PostGIS#1]

**Comment by koen-m** [comment]
Yes. Can confirm. [Section 1/tompkins.zip.zip](https://github.com/PacktPublishing/Learning-the-FOSS4g-Stack-Spatial-SQL-with-Postgres-PostGIS/blob/master/Section%201/tompkins.zip.zip) seems to be a corrupt archive currently.

**Comment by engstrom-jr** [comment]
Same issue. The zip file is corrupted. Just post the full tompkins.backup file, it is barely larger than the zip.

**Comment by engstrom-jr** [comment]
I found a workaround! there is a similar course that uses tompkins.zip here: https://github.com/PacktPublishing/Learning-the-FOSS4g-Stack-Python-for-Geospatial
